### PR TITLE
docs: revert docs migrated to templates [skip chromatic]

### PR DESCRIPTION
**Pull request by auroraVasconcelos**
## Description:
Closes https://github.com/solid-design-system/solid/issues/1420

## Definition of Reviewable:

- [x] Documentation is created/updated
- [x] Stories (features, a11y) are created/updated

### CODE REVIEW

File: packages/components/src/components/tab/tab.stories.ts [hunk 1]
```diff
@@ -55,9 +55,36 @@ export const Default = {
 
 export const Variant = {
   render: () => html`
-    <div class="flex gap-12">
-      <sd-tab variant="default">Default</sd-tab>
-      <sd-tab variant="container">Container</sd-tab>
+    <div class="grid grid-cols-2 gap-12">
+      <sd-tab-group activation="">
```
Comment (smfonseca) [inline packages/components/src/components/tab/tab.stories.ts:59]:
```suggestion
      <sd-tab-group>
```

File: packages/components/src/components/tab/tab.stories.ts [hunk 2]
```diff
@@ -55,9 +55,36 @@ export const Default = {
...
+        <sd-tab-panel name="tab-2">
+          <div>Provident illo neque vel ex.</div>
+        </sd-tab-panel>
+        <sd-tab slot="nav" panel="tab-3" variant="default">Default 3</sd-tab>
+        <sd-tab-panel name="tab-3">
+          <div>Provident illo neque vel ex. Inventore perspiciatis delectus nisi doloremque soluta inventore.</div>
+        </sd-tab-panel>
+      </sd-tab-group>
+
+      <sd-tab-group activation="">
```
Comment (smfonseca) [inline packages/components/src/components/tab/tab.stories.ts:74]:
```suggestion
      <sd-tab-group>
```

**Review by smfonseca — APPROVED** [review]

File: packages/components/src/components/tab/tab.stories.ts [hunk 3]
```diff
@@ -68,9 +95,36 @@ export const Variant = {
 
 export const Active = {
   render: () => html`
-    <div class="flex gap-12">
-      <sd-tab active>Default</sd-tab>
-      <sd-tab variant="container" active>Container</sd-tab>
+    <div class="grid grid-cols-2 gap-12">
```
Comment (karlbaumhauer) [inline packages/components/src/components/tab/tab.stories.ts:98]: can we have have the second tab activated? Now it actually looks exactly the same as the story above and therewith its not directly clear to the users, what "active" is actually doing. 
image

**Review by karlbaumhauer — CHANGES_REQUESTED** [review]

File: packages/components/src/components/tab/tab.stories.ts (hunk 3 above)
Comment (auroraVasconcelos) [reply packages/components/src/components/tab/tab.stories.ts:98]: @karlbaumhauer I totally agree with you and that was my first approach but for some reason it just ignores the "active" on second tab and makes the first active

Comment (karlbaumhauer) [reply packages/components/src/components/tab/tab.stories.ts:98]: thats strange. Pls have a look into this bug.

Comment (auroraVasconcelos) [reply packages/components/src/components/tab/tab.stories.ts]: Fixed. There is still an issue but a bugfix ticket for that is created: https://github.com/solid-design-system/solid/issues/1455.

File: packages/components/src/components/tab/tab.stories.ts [hunk 4]
```diff
@@ -91,41 +91,40 @@ export const Variant = {
 
 /**
  * Use the `active` attribute to toggle the active state.
+ *
+ * __Hint:__ Due to an storybook limitation it is necessary if the active tab is the first tab, please refresh the page.
```
Comment (karlbaumhauer) [inline packages/components/src/components/tab/tab.stories.ts:95]:
```suggestion
 * __Hint:__ Due to a Storybook issue, the first tab may be displayed as active. In this case, please reload the page. 
```

**Review by karlbaumhauer — APPROVED** [review]
approved besides one small text suggestion.

File: packages/components/src/components/tab/tab.stories.ts [hunk 5]
```diff
@@ -55,23 +55,76 @@ export const Default = {
 
 export const Variant = {
```
Comment (MartaPintoTeixeira) [inline packages/components/src/components/tab/tab.stories.ts:56]: For the variants when tab is open can you use the same copy on both default and container variants?
- Provident illo neque vel ex. Inventore perspiciatis delectus nisi doloremque soluta inventore.
- Lorem ipsum dolor sit amet, consectetur adipiscing elit.
- Inventore perspiciatis delectus nisi doloremque soluta inventore.

Comment (MartaPintoTeixeira) [reply packages/components/src/components/tab/tab.stories.ts:56]: Lets also apply this order to the rest of examples to reinforce that tabs as changing please

Comment (auroraVasconcelos) [reply packages/components/src/components/tab/tab.stories.ts:56]: Updated

**Review by MartaPintoTeixeira — APPROVED** [review]

**Pull request merged**